addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiClientSocket.java

```diff
@@ -74,9 +74,13 @@ public synchronized void open() throws Exception {
         if (isConnected()) {
             logger.warn("connect: connection is already open");
         }
-        if (!client.isStarted()) {
-            client.start();
-        }
+        // if (!client.isStarted()) {
```

Comment:
Not need, right?

Reply:
Indeed, you can trust the framework to start the common client for you